scss/components/_top-bar.scss

```diff
@@ -155,8 +155,8 @@ $topbar-unstack-breakpoint: medium !default;
   @else {
     .top-bar-title {
       float: left;
-      margin-right: $topbar-title-spacing;
-
+      margin: $topbar-title-spacing;
+      display:inline-block;
```

Comment:
@ncoden - The change has been fixed but just out of curiosity, were there any public discussions on the topic of display order for Foundation? I would be interested in reading what made this decision & what the pros/cons are to see if they are something my team might be interested in adopting in our projects.